src/Nancy/ErrorHandling/DefaultStatusCodeHandler.cs

```diff
@@ -62,47 +66,60 @@ public void Handle(HttpStatusCode statusCode, NancyContext context)
                 return;
             }
 
-            string errorPage;
-
-            if (!this.errorPages.TryGetValue(statusCode, out errorPage))
+            if (!this.errorMessages.ContainsKey(statusCode) || !this.errorPages.ContainsKey(statusCode))

```

Comment:
Sorry. I don't understand that. How is this protecting against calling without first checking the status code?
I read it as ensuring that there is always a message _and_ an error page for the status code.